sentry_sdk/client.py

```diff
@@ -143,6 +147,24 @@ def _get_options(*args, **kwargs):
     return rv
 
 
+def _update_scope(base, scope_change, scope_kwargs):
```

Comment:
We could move this to `scope.py` since it technically has nothing to do with the client

Reply:
Moved it, lets see if the CI checks are happy with it.

Reply:
looks good